torch/csrc/autograd/init.cpp

```diff
@@ -35,6 +35,14 @@ PyObject* THPAutograd_initExtension(PyObject* _unused, PyObject *unused) {
   auto _C_m = py::handle(torch_C_module).cast<py::module>();
   auto m = _C_m.def_submodule("_autograd", "autograd bindings");
 
+  auto parameter_module = THPObjectPtr(PyImport_ImportModule("torch.nn.parameter"));
```

Comment:
Doesn't really matter much, but out of curiosity, is this a `py::object`?

Reply:
It looks like there's an analog from TH. (Based on some grep-ing it seems to just be handling the incref and decref.) I just cargo cult'd the other ones.